### DISCUSSION TITLE
Wind speed input data in the LES model

### DISCUSSION THREAD

**Discussion by HEROISCOMING1** [firemodels/fds#13747] [Q&A] [answered]
Hi~~
When setting an area with a z grid of 2m, if the wind speed is input as follows, I understand that the input value is applied at an altitude of 2m and calculated by the wind speed profile equation above it, and I want to check whether I think it is correct.

"&WIND SPEED=10, DIRECTION=90, Z_0=1.0, L=-350. /"

Thank you~!

**Comment by mcgratta** [comment, marked as answer]
We recently changed the default reference height to 10 m. To be on the safe side, specify `Z_REF` explicitly. Then use the `POINTS` feature of a `DEVC` to produce a vertical profile of `QUANTITY='VELOCITY``.

**Reply by HEROISCOMING1** [reply]
Thank you for your answer.

I'm a beginner in fds, so I'm lacking a lot.
If so, can I write "&WIND SPEED=10, DIRECTION=90, Z_0=1.0, L=-350, Z_REF=00. /" to specify the location where the wind speed is input?

**Reply by mcgratta** [reply]
```
Z_REF=00.
```
What is the meaning of this? Do you want the wind speed to be 10 m/s right at the ground?

**Reply by HEROISCOMING1** [reply]
I would enter the altitude I wanted in place of '00'. 
For example, if the wind speed is 10m/s in the model area of ​​25m, 
I tried to write "&WIND SPEED=10, DIRECTION=90, Z_0=1.0, L=-350, Z_REF=25. /". 
Is this correct?

**Reply by mcgratta** [reply]
You mean to say a height of 25 m. Yes, this is correct.

**Reply by HEROISCOMING1** [reply]
Thank you so much!
